### ISSUE TITLE
[x86][Windows] External disassembler is not available.

### ISSUE THREAD

**Issue by sandreenko** [dotnet/coreclr#25131]
All last windows x86 JitStress jobs failed with: `External disassembler is not available`.

https://dev.azure.com/dnceng/public/_build/results?buildId=221425&view=ms.vss-test-web.build-test-results-tab

We did not have this problem in the previous run. Probably all GCStress=0xc jobs will fail as well.

It can be that the last helix update again deleted some necessary libraries from these machines (last time it was MSVC++ Redistributable dll, it was tracked in https://github.com/dotnet/core-eng/issues/6109).

I will check what we are missing now.

cc @MattGal @echesakov.

**Comment by sandreenko** [comment]
The problem was the missing package:
```
Running: dotnet restore "F:\workspace\_work\1\s\tests\\src\Common\stress_dependencies\stress_dependencies.csproj" --source https://dotnet.myget.org/F/dotnet-core/ --packages "F:\workspace\_work\1\s\tests\..\Packages"
F:\workspace\_work\1\s\tests\src\Common\stress_dependencies\stress_dependencies.csproj : error NU1101: Unable to find package runtime.ubuntu.14.04-x86.Microsoft.NETCore.CoreDisTools. No packages exist with this id in source(s): https://dotnet.myget.org/F/dotnet-core/
F:\workspace\_work\1\s\tests\src\Common\stress_dependencies\stress_dependencies.csproj : error NU1101: Unable to find package runtime.win-x64.Microsoft.NETCore.CoreDisTools. No packages exist with this id in source(s): https://dotnet.myget.org/F/dotnet-core/
F:\workspace\_work\1\s\tests\src\Common\stress_dependencies\stress_dependencies.csproj : error NU1101: Unable to find package runtime.win-x86.Microsoft.NETCore.CoreDisTools. No packages exist with this id in source(s): https://dotnet.myget.org/F/dotnet-core/
```
but the other jobs did not have this error (x64 from the same run was able to download that, and the next x86 in coreclr-ci were able to find it as well). 

@MattGal do you know what could happen in this ["build-tests](https://dev.azure.com/dnceng/public/_build/results?buildId=221425)" task?

**Comment by VincentBu** [comment]
Failed in below config:
[Windows.10.Amd64.Open-x86-Checked-tailcallstress](https://mc.dot.net/#/user/coreclr-outerloop-jitstress/ci~2Fdotnet~2Fcoreclr~2Frefs~2Fheads~2Fmaster/test~2Ffunctional~2Fcli~2F/20190611.1/workItem/JIT.Regression/analysis/xunit/JIT_Regression._JitBlue_GitHub_23199_GitHub_23199_GitHub_23199_~2F_JitBlue_GitHub_23199_GitHub_23199_GitHub_23199_cmd)

**Comment by MattGal** [comment]
@sandreenko yes I think I understand this, two points:

- Most (all?) teams after the 2.x time frame have stopped pushing their packages to myget.  If your team is still doing so (I'm unaware of such;  but it's possible) then you're likely to hit the problem that the storage capacity of the https://dotnet.myget.org/F/dotnet-core/ feed is full, which is why most teams moved to using the sleet style feeds instead.
- Your build should really, very much, fail if errors are thrown to prevent sending Helix work that can't succeed.

**Comment by sandreenko** [comment]
> Most (all?) teams after the 2.x time frame have stopped pushing their packages to myget. If your team is still doing so (I'm unaware of such; but it's possible) then you're likely to hit the problem that the storage capacity of the https://dotnet.myget.org/F/dotnet-core/ feed is full, which is why most teams moved to using the sleet style feeds instead.

We do not publish CoreDisTools there. I am not sure about other packages.

> Your build should really, very much, fail if errors are thrown to prevent sending Helix work that can't succeed.
I have a PR for it #25140.

**Comment by sandreenko** [comment]
Part of the issue was resolved by #25140, I will reopen it if it fails again because of infrastructure issues.
